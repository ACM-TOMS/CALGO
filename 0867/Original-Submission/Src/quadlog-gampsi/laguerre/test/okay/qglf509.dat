### Numerical integration with qglqf()
###
### int(x^p * x^alpha * exp(-x), x = 0..infinity)
###
### Line format: 2 p alpha neval result relerr abserr
###
