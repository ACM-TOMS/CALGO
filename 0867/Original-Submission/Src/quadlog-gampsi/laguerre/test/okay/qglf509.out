  Test integral: int(x^p * x^alpha * exp(-x), x = 0..infinity)

  1 ulp =   1.93E-34

  Floating-point precision appears to be (1 - log2(ulp) =) 113 bits

  alpha =  -2.000000  nquad =      1

  alpha =   1.000000  nquad =  32767

  Done
